src/index.tsx

```diff
@@ -6,6 +6,8 @@ import BugsnagPluginReact from "@bugsnag/plugin-react";
 import LogRocket from "logrocket";
 // eslint-disable-next-line no-restricted-imports
 import mixpanel from "mixpanel-browser";
+import dayjs from "dayjs";
+import relativeTimePlugin from "dayjs/plugin/relativeTime";
```

Comment:
The dayjs has this flow of extending its functionality. `RelativeTimePlugin` enables the functionality of using dayjs.fromNow(), which will return a nice string with time, spelled in letters

Reply:
Yup, aware of the pattern for importing it's 'extensions'

Where are we using `fromNow()`? Looks like just in 1 place?:

- `src/components/organisms/ChatSidebar/components/PrivateChats/components/PrivateChatPreview/PrivateChatPreview.tsx` 

Is there any reason we can't use `date-fns` `formatDistanceToNow` there?

- https://date-fns.org/
- https://date-fns.org/v2.17.0/docs/formatDistanceToNow
- https://date-fns.org/v2.17.0/docs/formatDistance

`date-fns` is a small/clean/modular, side effect free date library, and my preference for us to use (in most if not all places), though we definitely still have a lot of legacy stuff around that doesn't use it yet.

Feels like a more 'localised' change that needing to add dayjs plugins like this.

Further Reading:

- https://day.js.org/docs/en/display/from-now

Reply:
> `date-fns is a small/clean/modular, side effect free date library, and my preference for us to use (in most if not all places), though we definitely still have a lot of legacy stuff around that doesn't use it yet.`

I don't mind introducing new libraries, but I'd do it in a controlled/smooth way. Introducing them in such PRs doesn't seem like a good approach to me. Probably, we should discuss it and if we agree, we could create a PR, replacing the smelly code with this one. 

Feels like a more 'localised' change that needing to add dayjs plugins like this. I think, when you extend the `dayjs` lib., it changes it in every place used, so if that is the case, extending it in index.js seems like a good idea

Reply:
It's not introducing a new lib, it's a lib we already have/use in a number of places.

Reply:
>  think, when you extend the dayjs lib., it changes it in every place used, so if that is the case, extending it in index.js seems like a good idea

Yup, index is the correct place for 'extensions' like this. But my point was that libraries that use extensions like this are inherently bad, and if they require extensions, that's an antipattern we want to avoid as much as possible.